inst/project_templates/leej22_template_files/01_data_setup.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "{{folder_name}}"
+subtitle: "Data Setup"
+author: "Jasme Lee"
+date: "`r format(Sys.Date(), format = '%B %d, %Y')`"
+output:
+  html_document:
+    toc: true
+    toc_float: true
+    toc_depth: 2
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+
+set.seed({{format(Sys.Date(), format = "%Y%m%d")}})
+```
+
+# Raw Data Import
+
+```{r import}
+library(biostatR)
+library(tidyverse)
+library(labelled)
+library(lubridate)
+library(glue)
+
+# Set custom gtsummary theme if exists in the bstfun::theme_gtsummary_msk
+bstfun::theme_gtsummary_msk('leej22')
+
+# example code for importing excel file
+df_raw_data <-
+  readxl::read_excel(
+    path = here_data("Raw Data from PI.xlsx")
+  ) %>%
+  janitor::clean_names(case = "all_caps") %>% # uppercase col names
+  mutate(across(where(lubridate::is.POSIXt), lubridate::as_date)) # use lubridates
+```
+
+# Create Master Analytic Data Set
+
+```{r cleaning}
+
+```
+
+
+# Check Variables
+
+```{r checking}
+
+```
+
+# Save Analytic Data
+
+```{r save}
+# applying variable labels, and dropping raw variables
+df_main <- 
+  df_main %>%
+  labelled::set_variable_labels(
+    mrn = "Patient MRN"
+  )
+
+
+# saving master analytic data set
+saveRDS(
+  object = df_main,
+  file = here_data("dat.Rds")
```

Comment:
typically i try to merge all data together into 1 df (typically work with smaller datasets so it's not a problem) and save out the main data in dat . otherwise, i would save out multiple dfs in an .rdata file with the specific dfs named descriptively